cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -4,13 +4,21 @@ kind: Template
 metadata:
   name: cluster-service-admin
 parameters:
-- name: NAMESPACE
-  description: The namespace to create
-  required: true
-  value: cluster-service-admin
-- name: CLIENT_ID
-  description: The Azure Client ID used for federation
-  required: true
+  - name: NAMESPACE
+    description: The namespace to create
+    required: true
+    value: cluster-service-admin
+  - name: CLIENT_ID
+    description: The Azure Client ID used for federation
+    required: true
+  - name: MGMT_ORPHANED_NAMESPACE
+    description: The namespace to create to have a cronjob which will delete the orphaned namespace which are not deleted due to any issues with the jenkins job.
+    required: true
+    value: orphaned-namepsace-mgmt
+  - name: KUBECTL-IMAGE
```

Comment:
nit: lowercase for consistency with the other env vars

Reply:
Okay.